File: PHP/1-SRP/SRP-Solucao/Cliente.php
<?php
require_once('EmailServices.php');
require_once('CPFServices.php');

class Cliente
{
    public $id;
    public $nome;
    public $email;
    public $cpf;

    public function isValid()
    {
        $emailServices = new EmailServices();
        $cpfServices = new CPFServices();
        
        return $emailServices->isValid($this->email) && $cpfServices->isValid($this->cpf);
    }
}

File: Python/1-SRP/SRP-Solucao/Cliente.py
from EmailService import EmailService
from CPFService import CPFService

class Cliente:
    def __init__(self, id, nome, email, cpf):
        self.id = id
        self.nome = nome 
        self.email = email
        self.cpf = cpf
    
    def isValid(self):
        return EmailService().isValid(self.email) and CPFService().isValid(self.cpf)
